cran-comments.md

```diff
@@ -0,0 +1,5 @@
+## R CMD check results
```

Comment:
please could you remove this file as we do not need it (it has been removed in a previous PR but your branch was created when it had not been removed yet)